coverage_comment/main.py

```diff
@@ -273,6 +278,11 @@ def save_coverage_data_files(
 
     markdown_report = coverage_module.generate_coverage_markdown()
 
+    github.add_job_summary(
+        content=f"## Coverage report\n{markdown_report}",
```

Comment:
```suggestion
        content=f"## Coverage report\n\n{markdown_report}",
```
(I think I'm used to adding an empty line after titles in markdown)